Apps/Playground/ios/Podfile

```diff
@@ -4,24 +4,20 @@ require_relative '../node_modules/@react-native-community/cli-platform-ios/nativ
 platform :ios, '12.0'
 
 target 'Playground' do
-  permissions_path = '../node_modules/react-native-permissions/ios'
-  pod 'Permission-Camera', :path => "#{permissions_path}/Camera.podspec"
```

Comment:
We need these lines (except with react-native-permissions 3.0 we need to remove the `.podspec`).